### ISSUE TITLE
Simple key mapping doesn't work

### ISSUE THREAD

**Issue by Soromeister** [koekeishiya/skhd#361]
I am trying to do a very simple key remapping, basically remap `§` to ` (tick) and get the buttons inverted.

I tried the below configs:
```
0x0A : 0x32
```
```
0x0A = 0x32
```
```
0x0A -> 0x32
```

Nothing works so I'm opening an issue here to ask if this is even possible and if so, please provide examples as the current documentation is very ambiguous and completely lacks examples.

**Comment by tjex** [comment]
I think you may need to be looking at `skhd -k`, to simulate a keypress of another key, when pressing a certain key:

```text
a : skhd -k "f"
f <- I typed this by pressing 'a'
```

**Comment by Tenshikun** [comment]
I try to setup all my keyboard related config only using skhd, but when I try to swap two keys (lets say "a" and "b" for example) by doing : 
`a : skhd -k "b";
b : skhd -k "a"`
It doesn't work, it seems like it producing a endless loop or something like when I press "a" it sends "b" but then "b" is interpreted as an actual key pressing (which I know is the normal behaviour according to the help) but then the mapping "b -> a" triggers and the loop is back to the initial state, so nothing append when I press any of these two keys.

Does somebody knows how to bypass this please ?
Thanks a lot !

**Comment by tjex** [comment]
but... it's impossible for this to be any other way or? If `a = b` and `b = a`, then ... pressing `a` will execute `b`, which will execute `a`, which will execute `b`. This is a loop by nature. Or am I missing something?

**Comment by Tenshikun** [comment]
Yes but I mean I am looking for an alternative to produce the swapping keys but in a non-recursive way (the output of one key press is not considered by the input or the other one).

In many apps when it comed to key mapping you have the choice to specify wich behavior you chose between recursive and non recursive.

Probably there is a trucky way to do this with skhd but I just didn't find it so I wanted to know if somebody did it in the past.
